lib/smoke-test.js

```diff
@@ -189,14 +193,16 @@ function testUrls (opts) {
 	return directly(opts.throttle || 5, fetchers);
 }
 
-
 function runTest (opts) {
 	const appName = (opts.app) ? opts.app : 'ft-next-' + normalizeName(packageJson.name);
 	const urlConfig = require(path.join(process.cwd(), opts.configPath || 'test/smoke.js'));
 	baseUrl = 'http://' + appName;
 	if (!/:|\./.test(appName)) {
 		baseUrl += '.herokuapp.com';
 	}
+	if (opts.authenticated) {
+		authenticate = true;
```

Comment:
rather than globally setting, doesn't the `UrlTest` constructor have access to this `opts`, and therefore the `authenticated` flag?